### ISSUE TITLE
Jessica's Plan for the rest of Summer

### ISSUE THREAD

**Issue by Lilisys** [clarityinc/clarityeco#43]
7/20 - 24
- [x] fix client/img service sending the data
- [x] implement standard command args in tonic services
- [x] get rid of template folders across services 
  - [x] fix folder references and directory structure 
- [x] command arg documentation (wiki)
- [x] start cc documentation (wiki)
  - [ ] CC Readme (where does this go?)
- [x] Write Djinn + Tonic documentation (wiki)
- [x] condense img and djinn PR's down to 1
- [x] start resizing IPAService

7/27 - 31
- [x] Clean up IMGService
- [x] Remove local code from IMGService
- [x] IMGService documentation (wiki) 
- [x] Finish + Test the resizing service
  - [x] ~~cropping~~
  - [x] ~~turn to grayscale~~
- [x] Resizing service documentation (wiki)
- [x] ~~Test DjiNN with latest version of Caffe  (**in progress**)~~
- [x] Ensure IMGService works with IPA-Caffe

8/3 - 7
- [x] Write up about what I did this summer
- [ ] anything else I need to write up / logistics I'm forgetting

**Comment by jhauswald** [comment]
Says you're done on D&T documentation but if I check the wiki it looks incomplete on see the page about ....? It'd be good to link the djinn doc to the other services, makes navigating around intuitive. I like how you're making them standard with QuerySpec at the end.

**Comment by Lilisys** [comment]
My bad, just fixed it. Is there anything else I need to add there?
Also, for the tonic image wiki, I'm not sure about exactly how the page should be structured. I looked at @VivienZhou's [NLP Wiki page](https://github.com/clarityinc/clarityeco/wiki/Tonic-Natural-Language-Processing) and she wrote about how it was based off of Senna. The image service isn't based off a specific library is it? Should I just write a bit about image processing in general, and how neural networks are a part of it? Or is there some part of your paper I should quote?

**Comment by jhauswald** [comment]
Looks like you're almost done, I'm going through all your PRs and documentation. I had an issue with compiling tonic-img and thrift (odd caffe stuff) so we'll look at it tomorrow, probably a version thing. I noticed in a lot of the documentation its pretty vague about versions. We need to be careful for this in general.

other than that documentation looks good.

**Comment by Lilisys** [comment]
I'll add notes about versions to the documentations then, thanks for pointing that out. I started trying to install the newest version of Caffe, but got stuck because I ran into several 'undefined reference' issues to things that should be in caffe (SGDSolver, timer, net). I'll take a look at it some more today.

Also, where does the CC Readme go? I'm not actually sure about how to build it from scratch (i've been using an already built version for testing) so I won't be able to add that to the readme. I can link to some of the other services though, if that'd be helpful. If not, I'll just cross this item off my list.
